smoketests/pages/android_feedback_form.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from pages.feedback import FeedbackPage
+
+
+class AndroidFeedbackFormPage(FeedbackPage):
+    _page_title = 'Firefox Feedback :: Firefox Input'
+
+    def go_to_feedback_page(self,
+                            version="",
+                            channel="",
+                            last_visited="",
+                            in_device=False):
+        url = self.base_url + '/feedback/android'
+        params = []
+
+        if version:
+            url = '%s/%s' % (url, version)
+
+        if channel:
+            url = '%s/%s' % (url, channel)
+
+        if in_device:
+            params.append('utm_source=feedback-prompt')
+
+        if last_visited:
+            params.append('url=%s' % (last_visited))
+
+        if len(params) > 0:
+            url = '%s/?%s' % (url, '&'.join(params))
+
+        print('Requesting url: %s' % url)
```

Comment:
You don't generally want to print during tests. Also `print` as a function isn't available in Python 2.7 without a special import.